ReactiveCocoa/Swift/Bag.swift

```diff
@@ -60,7 +60,7 @@ internal struct Bag<T> {
 	/// will reset all current identifiers to reclaim a contiguous set of
 	/// available identifiers for the future.
 	private mutating func reindex() {
-		for var i = 0; i < elements.count; i++ {
+		for i in 0..<elements.count {
```

Comment:
This is already better, but there's an even safer way of creating this `Range`: using [`indices`](http://swiftdoc.org/func/indices/).